src/components/overview/unresolvedInfoBox/index.tsx

```diff
@@ -0,0 +1,59 @@
+import { Overlay } from '../overlay/styled';
+import UnresolvedTicketsModal from '../modals/unresolvedTickets';
+import {
+  Container,
+  Group,
+  GroupContainer,
+  GroupName,
+  HeadingContainer,
+  HeadingText,
+  MainInfoContainer,
+  StatusName,
+  StyledButton,
+  StyledHr,
+  TicketsContainer,
+  Value,
+} from './styled';
+import { IUnresolvedTickets, tickets } from './helper';
+import React, { useState } from 'react';
+
+const UnresolvedInfoBox = () => {
+  const [modalOpen, setModalOpen] = useState(false);
+  const toggleModal = () => {
+    setModalOpen(!modalOpen);
+  };
+  const lastTicketsNumber: number = -4;
+  const slicedTickets: IUnresolvedTickets[] = tickets.slice(lastTicketsNumber);
+
+  return (
+    <Container>
```

Comment:
Do some indent between independent parts in JSX to make a code more readable

Reply:
fix: indentation added in unresolvedInfoBox and tasksBox